app/views/teams/index.html.erb

```diff
@@ -14,7 +13,6 @@
   <tbody>
     <% @teams.each do |team| %>
       <tr>
-        <td><%= link_to team.id, team_path(team) %></td>
```

Comment:
Since the table's “id” column contained links to the specific team pages before the column got removed, we may consider adding the link to the “name” column instead. Maybe a question for our PO @svitlanakalancha?